app/services/userService.js

```diff
@@ -22,6 +24,25 @@ exports.findUserByEmail = email => UserModel.findOne({ where: { email: formatEma
 
 exports.findUserById = id => UserModel.findByPk(id);
 
+exports.findTopWeetAuthor = async () => {
+  try {
+    const mostWordWeetAuthor = await WeetModel.findAll({
+      where: {},
+      order: [[sequelize.fn('length', sequelize.col('content')), 'DESC']]
+    });
+
+    if (!mostWordWeetAuthor[0]) return null;
+
+    const topUser = await UserModel.findByPk(mostWordWeetAuthor[0].userId);
+    if (!topUser) return false;
+
+    return topUser;
+  } catch (err) {
+    logger.info(`findTopWeetAuthor ${err.message}`);
+    return false;
```

Comment:
Why are you returning false?

Reply:
Now i'm returning the error. This is because eslint obligates me to return something.